pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -853,21 +852,8 @@ def _check_dasd_formats(self):
         return rc
 
     def _check_space_and_get_dialog(self, disks):
-        # Figure out if the existing disk labels will work on this platform
-        # you need to have at least one of the platform's labels in order for
-        # any of the free space to be useful.
-        disk_labels = set(disk.format.label_type for disk in disks
-                              if hasattr(disk.format, "label_type"))
-        platform_labels = set(DiskLabel.get_platform_label_types())
-        if disk_labels and platform_labels.isdisjoint(disk_labels):
-            disk_free = 0
-            fs_free = 0
-            log.debug("Need disklabel: %s have: %s", ", ".join(platform_labels),
-                                                     ", ".join(disk_labels))
```

Comment:
Is removing this from logs a good thing? I can see how it is "not nice" to have it logged in the storage code, but it seems an useful piece of information to have. On the other hand I am not sure how one would correlate the lack of space with the log message, except during debugging. If the user is not learning that "_No present disks satisfy labels needed for the platform to be installed._" then I can see how losing it is not really much of a loss. Make of that what you will...